environment/src/main/java/com/sequenceiq/environment/tags/service/DefaultInternalAccountTagService.java

```diff
@@ -100,6 +101,30 @@ public void validate(List<AccountTag> accountTags) {
                 throw new BadRequestException(
                         String.format("The value '%s' can only can not start with microsoft or azure or aws or windows or space", accountTag.getTagValue()));
             }
+            if (isAccountTagContainsTemplate(accountTag.getTagKey())) {
+                if (isAccountTagInvalid(accountTag.getTagKey())) {
```

Comment:
`isAccountTagContainsTemplate(accountTag.getTagKey()) && isAccountTagInvalid(accountTag.getTagKey())` is the same